Services/Cron/README.md

```diff
@@ -0,0 +1,58 @@
+# Cron
+
+**Table of Contents**
+* [ilCronJobResult](#ilCronJobResult)
+
+## ilCronJobResult
+
+The class `ilCronJobResult` determine the status
+of a current cron job.
+
+The status are:
+* `STATUS_INVALID_CONFIGURATION`
+  This status will indicate the current configuration
+  for this cron job is not correct and MUST be
+  adjusted.
+* `STATUS_NO_ACTION`
+  This status will indicate that the cron job
```

Comment:
This status will indicate that the cron job did not perform any action.

Examples
* A cron job sending emails decided to send no emails at all.
* A cron job responsible for deleting orphaned objects did not find any object to delete.
* The lucene cron job decided that the index does not require an update.
* etc.